client/sites-dashboard/components/sites-site-url.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+import { ReactNode, useState, AnchorHTMLAttributes } from 'react';
+import ExternalLink from 'calypso/components/external-link';
```

Comment:
Did you true the `ExternalLink` component from `@wordpress/components`? It's good to favour those components where possible.

Reply:
I did try that one, that doesn't have a icon property, the icon is always present I would have to use css to show/hide the icon